tests/groupby_test.py

```diff
@@ -179,8 +181,33 @@ def test_broadcast_ints(self):
         self.assertTrue((np.array([0,0,0,0,0,1,1,0,1,1]),results.to_ndarray()))     
         
         results = self.igb.broadcast(1*(counts < 4))
-        self.assertTrue((np.array([1,0,0,0,0,1,1,1,1,1]),results.to_ndarray()))  
-        
+        self.assertTrue((np.array([1,0,0,0,0,1,1,1,1,1]),results.to_ndarray()))
+
+    def test_broadcast_uints(self):
+        keys, counts = self.ugb.count()
+
+        self.assertTrue((np.array([1, 4, 2, 1, 2]) == counts.to_ndarray()).all())
+        self.assertTrue((np.array([1, 2, 3, 4, 5]) == keys.to_ndarray()).all())
+
+        results = self.ugb.broadcast(1 * (counts > 2))
+        self.assertTrue((np.array([0, 1, 1, 1, 1, 0, 0, 0, 0, 0]), results.to_ndarray()))
```

Comment:
Should these be `self.assertTrue((np.array(...) == results.to_ndarray()).all())`, here and on lines 196 and 199? As written, I think it is evaluating the truth value of a tuple.

Reply:
yeah I noticed this, the same issue is present in `test_broadcast_ints` and `test_broadcast_booleans`
https://github.com/Bears-R-Us/arkouda/blob/78d3104a1211de039e5e84a88d987c99fe6bc996/tests/groupby_test.py#L175-L182
https://github.com/Bears-R-Us/arkouda/blob/78d3104a1211de039e5e84a88d987c99fe6bc996/tests/groupby_test.py#L190-L197

When I try to change these to do `== .all()` they actually fail (int and bool too). I think I'm gonna put in an issue to track this

Reply:
This is now being tracked in issue https://github.com/Bears-R-Us/arkouda/issues/1286